### DISCUSSION TITLE
[Q] Disabled init preview

### DISCUSSION THREAD

**Discussion by timsofteng** [Aloxaf/fzf-tab#313] [Q&A]
Hello. How can I disable previewers by default and open it with some hotkey?

**Comment by Aloxaf** [comment]
You can look into [fzf's manual](https://man.archlinux.org/man/fzf.1)

```zsh
zstyle ':fzf-tab:complete:cd:*' fzf-preview 'exa -1 --color=always $realpath'
zstyle ':fzf-tab:complete:cd:*' fzf-bindings 'ctrl-/:toggle-preview'
zstyle ':fzf-tab:complete:cd:*' fzf-flags --preview-window hidden
```

Above config will hide preview window and toggle it with <kbd>Ctrl+/</kbd>
